packages/dcos-integration-test/extra/conftest.py

```diff
@@ -177,6 +177,15 @@ def _wait_for_srouter_slaves_endpoints(self):
             assert "id" in data
             assert data["id"] == slave_id
 
+    @retrying.retry(wait_fixed=2000,
+                    retry_on_result=lambda r: r is False,
+                    retry_on_exception=lambda _: False)
+    def _wait_for_metronome(self):
+        r = self.get('/service/metronome/v1/jobs')
+        if r.status_code == 500 or r.status_code == 504:
```

Comment:
My intention was to make this more specific than what is used in the fixture elsewhere:
```
# resp_code >= 500 -> backend is still down probably
if r.status_code <= 500:
            logging.info("service is probably up")
            return True
        else:
            # retry
```
Would you prefer to use this less specific check and confine the metronome specificity to an actual metronome test?